src/sections/landing.tsx

```diff
@@ -4,20 +4,25 @@ import { russoOneHeadline } from "../utils/font";
 import { BsCalendar, BsGeoAlt } from "react-icons/bs";
 import CTAList from "../components/cta_list";
 import CTAButton from "../components/cta_button";
+import Link from 'next/link'
 
 const Landing = () => {
   return (
     <header
       className="w-full bg-gradient-to-r from-blue from-70%"
       style={{
-        height: "600px"
-      }}
+        display: "flex",
+        flexDirection: "row",
+        justifyContent: "center",
+        alignItems: "center",
+        height: "100vh",
+        }}
     >
       <img
         className="absolute right-0 -z-10 object-cover"
```

Comment:
```suggestion
        className="h-screen absolute right-0 -z-10 object-cover"
```

Se o height vai ser 100% da tela, a gente pode usar a prop `h-screen` do tailwind, que faz a mesma coisa. Nao consegui colocar no commit, mas se fizer isso, vai ter que apagar a linha 25 tambem, onde ta `height: "100vh".